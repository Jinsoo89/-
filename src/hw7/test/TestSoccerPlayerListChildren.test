# Test to display list of children of node on soccer players graph

# Load soccerPlayers.tsv
LoadGraph players soccerPlayers.tsv

ListChildren players C.Ronaldo