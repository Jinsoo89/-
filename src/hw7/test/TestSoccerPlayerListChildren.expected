# Test to display list of children of node on soccer players graph

# Load soccerPlayers.tsv
loaded graph players

the children of C.Ronaldo in players are: D.Maria(0.333) Isco(1.000) L.Modric(0.500) S.Ramos(1.000) Z.Ibrahimovic(1.000)